app/views/admin/users/actions/_merge.html.erb

```diff
@@ -0,0 +1,15 @@
+<div id="merge-accounts">
+  <%= form_for(@user, url: merge_admin_user_path(@user), html: { class: "flex flex-col gap-4", method: :post, onsubmit: "return confirm('Are you sure? This is extremely destructive and irreversible. Merging will delete all the other users content and combine it with this user')" }) do |f| %>
+    <p>To merge a duplicate account, make sure you are currently on the page of the user you want to KEEP. Below, add the user id of the account to merge information from, and delete.</p>
```

Comment:
**_Quick, non-blocking suggestion:_** maybe we could bold the "KEEP" here rather than capitalizing it all? Alternatively, we could both bold and capitalize "KEEP" to really draw an Admin's attention to it.

I lean more towards strictly bolding it rater than both bolding and capitalizing it to keep it consistent with the "deleted" on [line 7](https://github.com/forem/forem/pull/16174/files#diff-841ff8e29feba11327f0cf82827bf5407cb39e87cb90487efec1c39440c2f9e9R7) in this file.
```suggestion
    <p>To merge a duplicate account, first make sure you are currently on the page of the user you want to <strong>keep<strong>. Below, add the user ID of the account to merge information from, and delete.</p>
```